netlink/netlink_test.go

```diff
@@ -285,6 +286,10 @@ func TestAddRemoveStaticArp(t *testing.T) {
 	mac, _ := net.ParseMAC("aa:b3:4d:5e:e2:4a")
 	nl := NewNetlink()
 
+	// wait for interface to fully come up
+	// if it isn't fully up it might wipe the arp entry we're about to add
+	time.Sleep(time.Millisecond * 100)
```

Comment:
Are we waiting for a specific state here? We should avoid this type of behavior and explicitly wait for a state of the netdev and timeout if the expected state is never seen.